src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -270,10 +297,16 @@ private static void setDbAdaptors(MySQLAdaptor dbAdaptor)
 
 	}
 
+	private static void readAndSetHomologueMappingFile(String species, String hsap, String pathToOrthopairs) throws IOException {
+		Map<String,String[]> homologueMappings = readHomologueMappingFile(species, "hsap", pathToOrthopairs);
+		ProteinCountUtility.setHomologueMappingFile(homologueMappings);
+		EWASInferrer.setHomologueMappingFile(homologueMappings);
+	}
+
```

Comment:
The second argument doesn't look like it's used in this method - can it be removed?

Reply:
Ahah, thats clearly a mistake on my part, seems like it may have come from automatically generating this method. I changed the second argument to 'fromSpecies' and had it used in the first line of the method